modules/ROOT/pages/bolt/message.adoc

```diff
@@ -399,13 +399,19 @@ The `routing::Dictionary(address::String)` field is added in version *4.1* and a
 ----
 extra::Dictionary(
   user_agent::String,
+  patch_bolt::List<String>,
   scheme::String,
   routing::Dictionary(address::String),
   …
 )
 ----
 
 ** The `user_agent` should conform to `"Name/Version"` for example `"Example/4.1.0"` (see link:https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/User-Agent[] for more information).
+** `patch_bolt` is version *4.4 only*.
```

Comment:
it's 4.3 and 4.4

Reply:
Was it ported to 4.3 servers?

Reply:
Checked the driver code: 4.3. Don't know about server @gjmwoods ?

Reply:
yes ported back to 4.3